receiver/k8sclusterreceiver/internal/deployment/doc.go

```diff
@@ -0,0 +1,9 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !windows
+// +build !windows
```

Comment:
Why excluding windows?

Reply:
no idea, that's what the hpa code does as well. We can remove.

Reply:
I don't see any reason to keep it